x-pack/plugins/observability/public/components/shared/exploratory_view/utils/utils.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { uniq } from 'lodash';
+import { ApmIndicesConfig } from '../../../../../common/typings';
+
+export function getApmDataViewTitle(apmIndicesConfig: ApmIndicesConfig) {
```

Comment:
What do you think about renaming this file to something more meaningful? `utils/utils.ts` doesn't give me a clue about what I can find in this file.

Also, even if it's a simple function, it would be good to unit test this file.

Reply:
generally happy to address suggestion like this, but this issue has been pending for so long and i am almost on the verge of vacation, so going to consider this optional and will merge the PR now :)